eng/cibuild.sh

```diff
@@ -13,4 +13,5 @@ while [[ -h $source ]]; do
 done
 scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
 
-. "$scriptroot/common/build.sh" --restore --build --test --pack --integrationTest --publish --ci $@
+# SourceBuildIntermediateNupkgRid indicates to source-build that portable intermediates should be restored (required for runtime dependency)
+. "$scriptroot/common/build.sh" --restore --build --test --pack --integrationTest --publish --ci /p:SourceBuildIntermediateNupkgRid="linux-x64" "$@"
```

Comment:
Having to specify /p:SourceBuildIntermediateNupkgRid="linux-x64" does not seem like the optimal e2e UX here.  Can't the ArPOW infrastructure infer this?

Reply:
Have relatively little experience with ArPOW, but sounds doable. It would take some time for it to flow in though, right?

Reply:
Per a discussion offline, @oleksandr-didyk is going to add infra into arcade to handle this so that repo's don't have to specify this.

Reply:
The functionality was added to the arcade infra but it currently cannot be flown-in due to new Arcade version bumping the .NET version to 8.0. Mentioned this in the relevant Maestro PR -> https://github.com/dotnet/sourcelink/pull/936#issuecomment-1411884460

Reply:
Removed with the new changes from Arcade (SB build logic will identify the RID param by itself)